### PR TITLE
Move to storybook 5

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Move to storybook 5

**Code changes:**

- Swap the readme addon for the notes one
- Update webpack config

---

### 'import/no-extraneous-dependencies': 'off'
So, awkward, turns out this never worked... If you go on latest and log `paths` on line 71 of  `node_modules/eslint-plugin-import/lib/rules/no-extraneous-dependencies.js` it logs 
```
[ '/Users/mcmild07/dev/psammead/package.json',
  '/Users/mcmild07/package.json' ]
```
instead of what we expected;
```
[ '/Users/mcmild07/dev/psammead/packages/components/psammead-image/package.json',
  '/Users/mcmild07/dev/psammead/package.json' ]
```
I will raise an issue to fix this properly (https://github.com/bbc/psammead/issues/340), if possible, but for now theres no harm disabling it as it pretty much already is

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -13,16 +13,25 @@ import {
...
-  url: 'https://github.com/bbc/psammead',
-  addonPanelInRight: true,
-  sidebarAnimations: true,
-  sortStoriesByKind: true,
+const theme = create({
+  base: 'light',
+  brandTitle: 'BBC Pssammead',
+  brandUrl: 'https://github.com/bbc/psammead',
+  brandImage:
+    'https://user-images.githubusercontent.com/11341355/54079666-af202780-42d8-11e9-9108-e47ea27fddc5.png',
```
Comment (dr3) [inline .storybook/config.js:25]: I have added this as without it you lose the word 'Psammead' from the top left

File: packages/components/psammead-brand/src/index.stories.jsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
```
Comment (sareh) [inline packages/components/psammead-brand/src/index.stories.jsx:1]: At the top of each README.md file, we link to the Storybook URL e.g. https://bbc.github.io/psammead/?selectedKind=Brand  In Storybook v5, the URL structure changes, so will need to be updated to this format: 
https://bbc.github.io/psammead/?path=/info/brand--default

File: .storybook/config.js [hunk 3]
```diff
@@ -13,16 +13,25 @@ import {
...
+});
+
+addParameters({
+  options: {
+    panelPosition: 'right',
+    sidebarAnimations: true,
+    sortStoriesByKind: true,
```
Comment (sareh) [inline .storybook/config.js]:
```suggestion
```
This does not appear to be sorting the stories. Likely due to the value of the separator not being added. Removing this parameter retains this order:

Screen Shot storybook menu

File: .storybook/config.js [hunk 4]
```diff
@@ -13,16 +13,25 @@ import {
...
+    theme,
+  },
 });
 
 const GlobalStyle = createGlobalStyle`
```
Comment (sareh) [inline .storybook/config.js:36]: I checked that Global Styles are loading correctly & that the AMP Decorator is loading for the AMP image component. 👍 All working great.

File: packages/components/psammead-brand/src/index.stories.jsx [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
 import { storiesOf } from '@storybook/react';
-import { withReadme } from 'storybook-readme';
 import Readme from '../README.md';
```
Comment (sareh) [inline packages/components/psammead-brand/src/index.stories.jsx]:
```suggestion
import notes from '../README.md';
```
By changing the name of the import, we could tidy up the notes declaration below.

File: packages/components/psammead-brand/src/index.stories.jsx [hunk 6]
```diff
@@ -1,9 +1,10 @@
...
 import Brand from './index';
 
-storiesOf('Brand', module)
-  .addDecorator(withReadme(Readme))
-  .add('default', () => <Brand brandName="Default Brand Name" />);
+storiesOf('Brand', module).add(
+  'default',
+  () => <Brand brandName="Default Brand Name" />,
+  { notes: Readme },
```
Comment (sareh) [inline packages/components/psammead-brand/src/index.stories.jsx]:
```suggestion
  { notes },
```

File: packages/components/psammead-brand/src/index.stories.jsx (hunk 2 above)
Comment (sareh) [reply packages/components/psammead-brand/src/index.stories.jsx:1]: To ensure the Readme updates are pushed up to npm, we'll need to bump all of the package versions. Please note updating them in `package.json` and `package-lock.json` as well as adding notes in the `CHANGELOG.md`

**Review by sareh — CHANGES_REQUESTED** [review]
This looks great locally. ✨ I've just added some comments about updating links in Readmes and tidying up the notes implementation.

File: packages/components/psammead-brand/src/index.stories.jsx (hunk 5 above)
Comment (dr3) [reply packages/components/psammead-brand/src/index.stories.jsx]: Done :)

File: packages/components/psammead-brand/src/index.stories.jsx (hunk 6 above)
Comment (dr3) [reply packages/components/psammead-brand/src/index.stories.jsx]: Done :)

File: .storybook/config.js (hunk 3 above)
Comment (dr3) [reply .storybook/config.js]: Done :)

File: packages/components/psammead-brand/src/index.stories.jsx (hunk 2 above)
Comment (dr3) [reply packages/components/psammead-brand/src/index.stories.jsx:1]: Done :)

File: packages/components/psammead-caption/README.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# psammead-caption - [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?selectedKind=Caption) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-caption.svg)](https://www.npmjs.com/package/@bbc/psammead-caption) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC/psammead/blob/latest/CONTRIBUTING.md)
+# psammead-caption - [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/caption--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-caption.svg)](https://www.npmjs.com/package/@bbc/psammead-caption) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC/psammead/blob/latest/CONTRIBUTING.md)
```
Comment (sareh) [inline packages/components/psammead-caption/README.md]:
```suggestion
# psammead-caption - [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/brand--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-caption.svg)](https://www.npmjs.com/package/@bbc/psammead-caption) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC/psammead/blob/latest/CONTRIBUTING.md)
```

File: packages/components/psammead-sitewide-links/package.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-sitewide-links",
-  "version": "0.3.0",
+  "version": "0.3.1s",
```
Comment (sareh) [inline packages/components/psammead-sitewide-links/package.json]:
```suggestion
  "version": "0.3.1",
```

File: packages/components/psammead-sitewide-links/package-lock.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-sitewide-links",
-  "version": "0.3.0",
+  "version": "0.3.1s",
```
Comment (sareh) [inline packages/components/psammead-sitewide-links/package-lock.json]:
```suggestion
  "version": "0.3.1",
```

File: packages/components/psammead-brand/README.md [hunk 10]
```diff
@@ -1,4 +1,4 @@
-# psammead-brand - [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?selectedKind=Brand) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-brand.svg)](https://www.npmjs.com/package/@bbc/psammead-brand) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+# psammead-brand - [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/info/imageplaceholder--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-brand.svg)](https://www.npmjs.com/package/@bbc/psammead-brand) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
```
Comment (sareh) [inline packages/components/psammead-brand/README.md]:
```suggestion
# psammead-brand - [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/brand--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-brand.svg)](https://www.npmjs.com/package/@bbc/psammead-brand) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
```

**Review by sareh — DISMISSED** [review]
👍 Looks great!

**Review by twinlensreflex — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]

**Review by twinlensreflex — DISMISSED** [review]

**Review by sareh — APPROVED** [review]
👍 Works great locally.

**Review by ChrisBAshton — APPROVED** [review]
Looks good to me!

**Comment by ChrisBAshton** [comment]
Does this PR solve #266?

**Comment by dr3** [comment]
@ChrisBAshton Nope

**Comment by dr3** [comment]
Tested locally. All acts as expected except for the amp stories which require a refresh, howeve that was the case before.

**Pull request merged**